datafusion/src/physical_plan/tdigest/mod.rs

```diff
@@ -266,7 +266,6 @@ impl TDigest {
     }
 
     fn merge_sorted_f64(&self, sorted_values: &[OrderedFloat<f64>]) -> TDigest {
-        #[cfg(debug_assertions)]
```

Comment:
Delete it temporarily to see if CI will report an error

Reply:
it works. 
![image](https://user-images.githubusercontent.com/41979257/152370347-03623bfc-8ef0-4173-ac86-985dacb98086.png)
cc @alamb